pom.xml

```diff
@@ -278,6 +278,18 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.19</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Does this introduce dependencies to all of aws-crt-java?

If so, is there any way we can structure this so the dependencies are only taken on by Elasticurl?
Like maybe Elasticurl should move out of tests, so it can have its own pom.xml file with its own dependencies?

Reply:
The test dependency will not be included during runtime. Should not be a concern when we ship aws-crt-java to customers. But, a really good call!
I also tested the previous release, the junit and mockito are not included as well:
```
jdeps -s aws-crt-0.15.9.jar 
aws-crt-0.15.9.jar -> java.base 
```